app/lib/utils/queryMapper.js

```diff
@@ -21,6 +21,43 @@ function getLocationHeading(query) {
   return undefined;
 }
 
+function getResultsHeading(query, loc) {
+  const location = loc.toUpperCase();
+  if (query.type) {
+    if (comparisons.areEqual(query.type, constants.serviceTypes.professional)
```

Comment:
Extract this check into a function to help understand what is being checked at a glance.